src/pages/home/components/ScrollTopButton.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import Button from 'components/button/Button';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
```

Comment:
We are now defining our own icons, check `components/icon`. Please don't use `fortawesome` anymore, we want to remove that dependency. 

Given that we don't have an arrow icon in our library, maybe just use the vectors from `fortawesome` to create our own icon, I didn't see an arrow icon in our set of icons here: https://sketch.cloud/s/xVR5A/ELzbv7q

Reply:
Good! I'll try to get  the icon from `fontawesome`. Because I have a busy week, I'll got this done for the next weekend! I hope this works for you. Thank you!

Reply:
Created a new icon in the component/icon folder named ArrowUp based on svg Icon component
On the ScrollTopButton component replaced `fontawesome` with the new ArrowUp button
I hope everything is fine in these changes!